pkgs/test_core/lib/backend.dart

```diff
@@ -11,3 +11,10 @@ export 'package:test_api/backend.dart'
     show Metadata, PlatformSelector, Runtime, SuitePlatform;
 
 export 'src/runner/parse_metadata.dart' show parseMetadata;
+
+export 'src/runner/configuration/load.dart' show load, loadFromString;
+export 'src/runner/suite.dart' show SuiteConfiguration;
+
+export 'src/runner/configuration.dart' show Configuration;
```

Comment:
`configuration.dart` already has a transitive import to `dart:io`, can we add `Configuration.load` and `Configuration.loadFromString` static methods (or factory constructors) to avoid the overly broad top level `load` name?

We can keep the implementation as is for now and add something small to forward them so it's not a large refactor.

Reply:
Sure, sgtm